test/TestExtensions/GenerateLicenseList/GenerateLicenseList.csproj

```diff
@@ -7,12 +7,12 @@
     <NoWarn>$(NoWarn);NU1505</NoWarn> <!-- Remove NoWarn when https://github.com/dotnet/sdk/issues/24747 is fixed -->
     <Description>A utility for updating the NuGet license list from the SPDX source.</Description>
     <!-- There are some conflicts that are impossible to resolve because of missing package versions-->
-    <MicrosoftCodeAnalysisCSharpVersion>3.0.0</MicrosoftCodeAnalysisCSharpVersion>
+    <MicrosoftCodeAnalysisVersion>3.0.0</MicrosoftCodeAnalysisVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
-    <PackageReference Include="Microsoft.CodeAnalysis" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
+    <PackageReference Include="Microsoft.CodeAnalysis" VersionOverride="$(MicrosoftCodeAnalysisVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" VersionOverride="$(MicrosoftCodeAnalysisVersion)" />
```

Comment:
Did you try removing the version override, and see if there are still problems? I don't think there are any good reasons for this project to use older analyers.

Reply:
There are breaking API changes. This project generates code, so uses APIs from these packages, idk how complicated fixing that would be. 

We should at least track it if we want to make it consistent eventually, but imo given that this is something that just generates some minimal amounts of code, I don't care too much about it using an updated version.